src/main/java/org/folio/des/security/AuthService.java

```diff
@@ -32,20 +35,28 @@ public SystemUserParameters loginSystemUser(String tenant, String url) {
             .password(username)
             .build();
 
-    log.info("Login with url={} tenant={} username={}.", url, tenant, username);
+    log.info("Attempt login with url={} tenant={} username={}.", url, tenant, username);
 
     ResponseEntity<String> authResponse = authClient.getApiKey(userParameters);
 
     var token = authResponse.getHeaders().get(XOkapiHeaders.TOKEN);
     if (isNotEmpty(token)) {
-      userParameters.setOkapiToken(token.get(0));
-      usersClient.getUsersByQuery("username==" + username).getUsers().stream().findFirst().ifPresentOrElse(user ->
-        userParameters.setUserId(user.getId()), () -> log.error("Can't find user id by username {}.", username));
       log.info("Logged in as {}.", username);
+      userParameters.setOkapiToken(token.get(0));
     } else {
       log.error("Can't get token logging in as {}.", username);
     }
-    return userParameters;
+    return userParameters.getOkapiToken();
+  }
+
+  public String getSystemUserId() {
+    Optional<User> optionalUser = usersClient.getUsersByQuery("username==" + username).getUsers().stream().findFirst();
+
+    if(optionalUser.isEmpty()) {
```

Comment:
missed space. Please, format the lines you've added

Reply:
Fixed